src/rgw/rgw_bucket_logging.cc

```diff
@@ -31,6 +31,7 @@ bool configuration::decode_xml(XMLObj* obj) {
       logging_type = LoggingType::Standard;
     } else if (type == "Journal") {
       logging_type = LoggingType::Journal;
+      RGWXMLDecoder::decode_xml("Filter", s3_filter, o);
```

Comment:
please check that we only got key filter, and throw an error if we got another type